themes/digital.gov/src/js/common/edit-this.js

```diff
@@ -22,12 +22,20 @@ jQuery(document).ready(function($) {
 			// remove the active class from the edit button
 			$('.edit_tools').removeClass('active');
 			// swap out the icon in the edit button
-			$(this).html('<i class="far fa-edit"></i>');
+			$(this).html('<i class="far fa-edit"></i><span class="screenreaderonly">Toggle edit this page</span>');
 		} else {
 			enable_edit_this();
 			$('.edit_tools').addClass('active');
-			$(this).html('<i class="fas fa-times"></i>');
+			$(this).html('<i class="fas fa-times"></i><span class="screenreaderonly">Toggle edit this page</span>');
 		}
   });
 
 });
+
+$(function(){
+    $('.edit-issue').keypress(function(e) {
+        if(e.keyCode == 32){
```

Comment:
Minor, but I would save this to a `const SPACEBAR = 32;` and then do something like 
```
if (e.keyCode == SPACEBAR) {
  …code
}
```

Or just add a comment mentioning spacebar.